### PR TITLE
Additions To Documentation

### PR DESCRIPTION
**Pull request by Arsenikr**
- split context standards to mandatory and recommended
- added some more best practices.

Please review and add insights

## Status
In Progress

### CODE REVIEW

**Comment by fvigo** [comment]
Merged to new sidebar to avoid conflicts

**Review by fvigo — COMMENTED** [review]
Thanks @Arsenikr ! 

I made a few changes in the sidebar structure (aligned to the latest changed we made), and moved the Generic IAM Commands document under Advanced, I don't think we need a specific section for it right now (we might add it later as we have more similar articles)

I think the IAM command should be completed with all the examples before merging this. If you don't think you can complete it quickly, I suggest to remove it from this PR, we can merge this one and readd that once it's ready.

**Comment by Arsenikr** [comment]
added more fixes and additional things. please review

File: docs/playbooks/playbooks-generic.md
Comment (fvigo) [inline docs/playbooks/playbooks-generic.md:13]: this link is not correct

File: docs/playbooks/playbooks-generic.md
Comment (fvigo) [inline docs/playbooks/playbooks-generic.md:12]: this link is not correct

**Review by fvigo — CHANGES_REQUESTED** [review]
partial review, found some broken links

File: docs/integrations/generic-commands-about.md
Comment (fvigo) [inline docs/integrations/generic-commands-about.md:11]: Can you be a little more specific in describing what these commands are and why are important, maybe with some examples?

Also is `using` documented somewhere in the product doc you can link to?

File: docs/integrations/context-standards-recommended.md
Comment (David-BMS) [inline docs/integrations/context-standards-recommended.md:16]: How about a ticket summary or description?

File: docs/concepts/use-cases.md
Comment (fvigo) [inline docs/concepts/use-cases.md:142]: We should remove this link until `generic-iam-commands` is complete

File: docs/integrations/context-standards-about.md
Comment (fvigo) [inline docs/integrations/context-standards-about.md:6]: suggest to add a link here to `../concepts/concepts#context-data`

File: docs/integrations/context-standards-mandatory.md
Comment (fvigo) [inline docs/integrations/context-standards-mandatory.md:7]: Worth mentioning what we mean by `File` and the differences from `InfoFile` below.

File: docs/integrations/context-standards-mandatory.md
Comment (fvigo) [inline docs/integrations/context-standards-mandatory.md:632]: We should mention the differences from `File` above.

File: docs/integrations/context-standards-recommended.md
Comment (fvigo) [inline docs/integrations/context-standards-recommended.md:20]: We should add the YAML if possible

File: docs/integrations/context-standards-recommended.md
Comment (fvigo) [inline docs/integrations/context-standards-recommended.md:161]: We should add the YAML if possible

File: docs/integrations/context-standards-recommended.md
Comment (fvigo) [inline docs/integrations/context-standards-recommended.md:178]: We should add the YAML if possible

File: docs/integrations/context-standards-recommended.md
Comment (fvigo) [inline docs/integrations/context-standards-recommended.md:202]: We should add the YAML if possible

File: docs/integrations/generic-commands-reputation.md
Comment (fvigo) [inline docs/integrations/generic-commands-reputation.md:12]:
```suggestion
## Generic Reputation Commands

If an integration returns reputation on specific indicator types, it must implement the commands according to these naming and functionality standards.
```

File: docs/integrations/generic-commands-reputation.md
Comment (fvigo) [inline docs/integrations/generic-commands-reputation.md:16]: We should mention the argument name as well, and that is mandatory and default, and that the command should return the `File` context standard with a link to the schema. Maybe add the YML of the command so they can easily copy and paste.

File: docs/integrations/generic-commands-reputation.md
Comment (fvigo) [inline docs/integrations/generic-commands-reputation.md:20]: same as above

File: docs/integrations/generic-commands-reputation.md
Comment (fvigo) [inline docs/integrations/generic-commands-reputation.md:24]: same as above

File: docs/integrations/generic-commands-reputation.md
Comment (fvigo) [inline docs/integrations/generic-commands-reputation.md:28]: same as above

File: docs/integrations/generic-commands-reputation.md
Comment (fvigo) [inline docs/integrations/generic-commands-reputation.md:32]: same as above

File: docs/integrations/generic-commands-reputation.md
Comment (fvigo) [inline docs/integrations/generic-commands-reputation.md:36]: same as above

**Review by fvigo — CHANGES_REQUESTED** [review]
Added a few comments, looks good overall.

File: docs/integrations/dbot.md
Comment (anara123) [inline docs/integrations/dbot.md:42]: I think the descriptions here are a bit incorrect.
1. The indicator do not exists in the product/integration.
2. The indicators in the product have no score/vector (only data)

File: docs/integrations/context-standards-mandatory.md
Comment (anara123) [inline docs/integrations/context-standards-mandatory.md:210]: You have duplicates in both files. Delete from here Event, Account and others

File: docs/integrations/generic-commands-about.md
Comment (Arsenikr) [reply docs/integrations/generic-commands-about.md:11]: no we don't afaik

File: docs/integrations/generic-commands-reputation.md
Comment (Arsenikr) [reply docs/integrations/generic-commands-reputation.md:16]: A YML here will be a copy paste of the context standards. maybe i can link to it instead.. WDYT?

File: docs/integrations/context-standards-mandatory.md
Comment (fvigo) [inline docs/integrations/context-standards-mandatory.md:7]:
```suggestion
The following is the format for a file. File here refers to the file indicator or a binary file that could potentially be malicious, and might be checked for reputation or sent to a sandbox. 
```

**Review by fvigo — CHANGES_REQUESTED** [review]
Just added some minor tweaks and I think it's good to go

File: docs/integrations/context-standards-mandatory.md
Comment (glicht) [inline docs/integrations/context-standards-mandatory.md:2]: We need to add a redirect to the `_redirects` file

**Review by glicht — CHANGES_REQUESTED** [review]
@Arsenikr there are a few empty yaml examples as reported here: https://github.com/demisto/content-docs/issues/346 

Please add these too.

File: docs/concepts/use-cases.md
Comment (Arsenikr) [reply docs/concepts/use-cases.md:142]: removed fully. will add when IAM design will be stable

File: docs/integrations/dbot.md
Comment (Arsenikr) [reply docs/integrations/dbot.md:42]: about 1 i tend to agree it is better. 
about 2 - there is a case that the vendor actually returns "Unknown" for the indicator

so maybe: 
1. The vendor returns an "Unknown" score for the indicator.
2. The indicator do not exists in the product/integration.
3. The indicators in the product have no score/vector (only data)

WDYT

File: docs/integrations/context-standards-mandatory.md
Comment (Arsenikr) [reply docs/integrations/context-standards-mandatory.md:2]: added one to context-standards-about

**Review by fvigo — APPROVED** [review]

**Comment by Arsenikr** [comment]
@kirbles19 please review

File: docs/concepts/use-cases.md
Comment (kirbles19) [inline docs/concepts/use-cases.md:135]: Should this be: ILM (Identify Lifecycle Management)?

File: docs/concepts/use-cases.md
Comment (kirbles19) [inline docs/concepts/use-cases.md:140]: Manage access to resources and applications.

File: docs/concepts/use-cases.md
Comment (kirbles19) [inline docs/concepts/use-cases.md:139]: - Block a user
- For change of password

File: docs/integrations/context-standards-mandatory.md
Comment (kirbles19) [inline docs/integrations/context-standards-mandatory.md:7]: In this context, file refers to the file indicator or binary file that might be malicious and can be checked for reputation, or sent to a sandbox.

**Comment by kirbles19** [comment]
@Arsenikr Done.

File: docs/concepts/use-cases.md
Comment (michalgold) [reply docs/concepts/use-cases.md:135]: The use case is "Identity and Access Management". We just gave a different more generic name to our pack.

**Comment by Arsenikr** [comment]
> @Arsenikr there are a few empty yaml examples as reported here: #346
> 
> Please add these too.

@glicht filled those. any additional changes? i want to close the PR

**Review by glicht — APPROVED** [review]
looks good

**Pull request merged**
